front/src/applications/stdcm/utils/__tests__/simulationReportSheet.spec.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  generateCodeNumber,
+  formatCreationDate,
+  getStopDurationTime,
+  extractSpeedLimit,
+  getOperationalPointsWithTimes,
+} from 'applications/stdcm/utils/formatSimulationReportSheet';
+
+import { suggestedOperationalPoints, simulation, departure_time } from './sampleData';
+
+describe('Utils simulation report sheet functions', () => {
+  test('should return a formatted string', () => {
```

Comment:
Maybe you can complete the TODO of this function to specify that, when the code will be generated based on stdcm inputs hash, we will need to adapt the test as well

Reply:
done

Reply:
done